### DISCUSSION TITLE
Translation using faster-whisper

### DISCUSSION THREAD

**Discussion by elia-ashraf** [SYSTRAN/faster-whisper#224] [Q&A] [answered]
I have used fatser-whisper for transcription and so far it has been really accurate and definetly faster than OpenAI's Whisper. I have not only transcribed English audios but also Russian, Italian etc. Although it is faster and more accurate with English, the performance with other languages is also pretty good.

Now, my question is, can faster-whisper also translate from X->EN or from language X-> language Y?

I have tried 
`model.translate("audio_file.wav", beam_size=5)`

instead of
`model.transcribe("audio_file.wav", beam_size=5)`

but it seems that 'WhisperModel' object has does not have a 'translate' attribute.

**Comment by guillaumekln** [comment, marked as answer]
Whisper can translate from language X to English with `task="translate"`:

```python
model.transcribe("audio_file.wav", task="translate")
```

However, translating language X to language Y is not possible.

**Reply by elia-ashraf** [reply]
Thank you so much for your quick response. 

Is it possible to translate it from EN->X?

**Reply by guillaumekln** [reply]
No, Whisper can only do X->EN.

**Reply by elia-ashraf** [reply]
So I tried something as a fluke and it worked!!

I had a non-English audio and I used:
`model.transcribe("audio.mp3", beam_size=5, language="de")`

And here's the result
```
WARNING:libav.mp3:Estimating duration from bitrate, this may be inaccurate

CPU times: user 115 ms, sys: 16 ms, total: 131 ms
Wall time: 121 ms
Detected language 'de' with probability 1.000000
 Er ist nicht so wie andere. Er ist aufmerksam, aufmerksam, er weiß Antworten auf jeden Frage.
 Er teilt meine Stimme für gesundes Essen und kulinarische Experimente. Er gibt hilfliche Vorschläge.
 Und wir gehen immer zusammen auf Produkte-Magazine
```

But it is only with the older Whisper model, which was large-v1.
